app/views/layouts/partners/application.html.erb

```diff
@@ -64,9 +64,11 @@
 
   <div class="content-wrapper">
     <% flash.each do |key, value| %>
+      <!-- Ensure value is a string and sanitized for HTML rendering -->
+      <% message = value.is_a?(Hash) ? value.to_s : sanitize(value, tags: %w(ul li)) %>
```

Comment:
If value is a hash, then you are just calling to_s on it. 
This results in a message that looks like `{ a: :b}` which seems like it isn't what you want. Are you trying to extract the message out of the hash?